ui/ducks/send/send.js

```diff
@@ -683,9 +698,17 @@ const slice = createSlice({
      * field and send state, then updates the draft transaction.
      */
     calculateGasTotal: (state) => {
-      state.gas.gasTotal = addHexPrefix(
-        calcGasTotal(state.gas.gasLimit, state.gas.gasPrice),
-      );
+      // use gasPrice as the multiplier if working with a LEGACY transaction
+      // otherwise use maxFeePerGas.
```

Comment:
Nit: The below logic is slightly out of sync with the comment here. I think the below logic would be more accurately reflected by the following comment:

```
// use maxFeePerGas as the multiplier if working with a FEE_MARKET transaction
// otherwise us gasPrice
```
(i.e. first describe what is being done in the specific case checked for the in the `if` block, and the describe everything the `else` block handles with `otherwise`)